### DISCUSSION TITLE
Is there a way to have borders and decorations?

### DISCUSSION THREAD

**Discussion by jasursadikov** [gokcehan/lf#1802] [Q&A] [answered]
Is there a way to have borders like here:
![image](https://github.com/user-attachments/assets/788077c8-40e9-4743-8c5a-1b8d5cc17c61)
And line numbers in previewer and have a borders between parenting directory and current directory?
I cannot find an information about this in wiki.

Thank you!

**Comment by jasursadikov** [comment]
Also displaying binary files with xxd would be much preferable!

**Comment by DusanLesan** [comment, marked as answer]
You can use `bat` program for that like:
lfrc:
```
set previewer ~/.config/lf/preview
set cleaner ~/.config/lf/cleaner
```
preview:
```shell
#!/bin/sh

case "$(file --dereference --brief --mime-type -- "$1")" in
	application/octet-stream) xxd "$1";;
	*) bat --color=always "$1";;
esac

exit 0
```
